migra/command.py

```diff
@@ -47,6 +47,13 @@ def parse_args(args):
         default=False,
         help="Also output privilege differences (ie. grant/revoke statements)",
     )
+    parser.add_argument(
+        "--encode-utf8",
```

Comment:
This is a good way to go, except let's change the argument name/variable to `force-utf8` rather than `encode-utf8`.

Also, can I ask you what setup you have that makes you need this? Perhaps in the future sometime we can detect bad setups explicitly and fix them automatically without needing the option.

Reply:
@djrobstep I had issue #13 when my code (say trigger, function, etc) contained Persian text e.g., `raise 'پیام خطا';`. I could address the issue by encoding the output to `utf-8`, and I thought it might be of help to those have the same issue.

Reply:
Are you on windows? What python version are you running? I haven't been able to replicate this so it would be good to know what systems and console configurations it is affecting.

Reply:
Thanks for approving the PR,
My dev environment is Ubuntu 14.04 with Python 2.7.14. I use it as a virtual machine hosted on Mac OSX. This is to encapsulate my dev/test stuff, keeping the Mac clean. I run the command through the SSH terminal.
FYI, the error raises only when I am trying to save the output to a file. Otherwise it works fine, i.e., I can see the Persian output on the terminal!
Please let me know if you need more information, e.g., screenshot or something...